src/main/java/eu/elision/marketplace/services/helpers/Mapper.java

```diff
@@ -1,62 +1,40 @@
 package eu.elision.marketplace.services.helpers;
 
 import eu.elision.marketplace.domain.product.category.Category;
+import eu.elision.marketplace.domain.product.category.attributes.DynamicAttribute;
+import eu.elision.marketplace.domain.product.category.attributes.PickListItem;
 import eu.elision.marketplace.web.dtos.CategoryDto;
 import eu.elision.marketplace.web.dtos.CategoryMakeDto;
+import eu.elision.marketplace.web.dtos.DynamicAttributeDto;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class Mapper
-{
-    private Mapper()
-    {
+public class Mapper {
+    private Mapper() {
     }
 
-    public static Category toCategory(CategoryMakeDto categoryMakeDto)
-    {
+    public static Category toCategory(CategoryMakeDto categoryMakeDto) {
         final Category category = new Category();
         category.setName(categoryMakeDto.name());
         return category;
     }
 
-    public static CategoryDto toCategoryDto(Category category)
-    {
+    public static CategoryDto toCategoryDto(Category category) {
         return new CategoryDto(category.getId(), category.getName(), new ArrayList<>());
```

Comment:
Why empty attributes? At rule 31 you use `category.getCharacteristics().stream().map(Mapper::toDynamicAttributeDto).toList()`